### PR TITLE
4418 Update Dialogs

### PR DESCRIPTION
**Pull request by Liliya-web**
open/close delay doesn't work for the moment. Please add the functionality when the bug is fixed on the Vuetify side: https://github.com/vuetifyjs/vuetify/issues/15938 
The 4418 task stays open

### CODE REVIEW

File: components/dialogs/DisabledDialog.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<template>
+    <v-row justify="center">
+        <v-dialog v-model="dialog" disabled=true max-width="290">
```
Comment (omega790) [inline components/dialogs/DisabledDialog.vue:3]: disabled=true -> disabled
We don't need explicitly specify 'true' in this case

File: components/dialogs/PersistentDialog.vue [hunk 2]
```diff
@@ -26,6 +26,33 @@
                 </v-card-actions>
             </v-card>
         </v-dialog>
+
+        <v-dialog v-model="dialog" persistent no-click-animation=true max-width="290">
```
Comment (omega790) [inline components/dialogs/PersistentDialog.vue:30]: Could you tell me please what property is responsible for that?

File: components/dialogs/DisabledDialog.vue (hunk 1 above)
Comment (Liliya-web) [reply components/dialogs/DisabledDialog.vue]: Fixed

File: components/dialogs/PersistentDialog.vue (hunk 2 above)
Comment (Liliya-web) [reply components/dialogs/PersistentDialog.vue]: If you use no-click-animation there is no animation when you click outside an open persistent dialog, if you open this dialog the first time after the page reloading. Applicable for persistent dialogs only

**Review by omega790 — APPROVED** [review]

**Pull request merged**